augur/application/db/models/augur_data.py

```diff
@@ -3358,3 +3358,31 @@ class PullRequestReviewMessageRef(Base):
     msg = relationship("Message")
     pr_review = relationship("PullRequestReview")
     repo = relationship("Repo")
+
+
+class RepoClone(Base):
```

Comment:
@ABrain7710  / @IsaacMilarky : Is this the right way?

Reply:
There should be a unique constraint on the repo_id if you plan on using postgres 'on conflict' inserts.

Reply:
I did this, and then I realized this is like "releases", or repo_info... we want to hold the historical record for the repos. There should not be any conflicts since the primary key is an autoincrement @IsaacMilarky